src/main/java/com/cvsgo/controller/ProductController.java

```diff
@@ -37,6 +40,21 @@ public SuccessResponse<ProductDetailResponseDto> readProduct(@LoginUser User use
         return SuccessResponse.from(productService.readProduct(user, productId));
     }
 
+    @PostMapping("/{productId}/like")
```

Comment:
API path는 복수형으로 쓰는 것이 좋을 것 같습니다!
https://learn.microsoft.com/ko-kr/azure/architecture/best-practices/api-design
```suggestion
    @PostMapping("/{productId}/likes")
```

Reply:
앗 수정한다는걸 깜빡했네요 감사합니다!